testing/softhsm/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Frank Felhoffer <silveraid@hackme.ca>
+# Maintainer: Frank Felhoffer <silveraid@hackme.ca>
+pkgname=softhsm
+pkgver=2.3.0
+pkgrel=0
+pkgdesc="cryptographic store accessible through a PKCS #11"
+url="http://www.softhsm.org/"
+arch="all"
+license="BSD2"
+depends="openssl"
+makedepends="openssl-dev automake autoconf libtool file"
```

Comment:
Does this build with `libressl-dev`? If not, can you please file a bug upstream? We want avoid openssl if possible.

Reply:
I was able to compile it with libressl without any problem.